content/cpp/concepts/strings/terms/substr/substr.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '.substr()'
+Description: 'Returns a portion of a string specified by a starting position and length.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The `.substr()` method returns a portion of a string specified by a starting position and length.
+
+## Syntax
+
+```pseudo
+basestring.substr(pos,len)
+```
+
+- `basestring` is the string whose substring is being returned.
```

Comment:
I noticed in #750 that we used `baseString` (capital "S") instead of lowercase `basestring`. Can we pick one of the other?